src/main.cpp

```diff
@@ -49,8 +49,6 @@ using namespace std;
 
 TConsole *  spDebugConsole = 0;
 
-extern Q_CORE_EXPORT int qt_ntfs_permission_lookup;
```

Comment:
I read the docs. We don't use it, if we ever do, adding this back will be the easiest part of the change.